content/api-streamer/overview.textile

```diff
@@ -0,0 +1,39 @@
+
+Typically today, data producers provide rate-limited REST APIs for developers to consume data from. But increasingly those APIs are failing to meet developer performance expectations, as polling a REST API offers substandard latencies. So data producers are creating realtime APIs to meet developer demand for realtime data.
+
+However, creating realtime APIs without a realtime API management platform in place to maintain and manage those APIs leads to higher complexity, cost and pressure on the engineering teams to implement these crucial features from scratch.
+
+This is the gap that API Streamer fills: ready-built API management tools and a distribution network for creators of realtime APIs.
+
+h2(#publishing-data-with-api-streamer). Managing your data streams with Ably's API Streamer
+
+If you are familiar with Ably's Data Stream Network (DSN), you know that it lets you create any number of applications to stream data in realtime to any of your clients. Using the API Streamer, you can share the same data streams with developer teams at other organizations, with clear and easy to use Realtime API management features such as consolidated stats and analytics, list view of all the consumers of your streams, rate limits and such.
+
+If you already have a DSN app, you can skip to learning about the "various elements of the API Streamer":#api-streamer-elements and how to set those up.
+
+<%= partial partial_version('/api-streamer/_dsn-app-setup') %>
+
+h2(#api-streamer-elements). Understanding the elements of the API Streamer
+
+With the API Streamer, you can either publish your realtime data streams via your realtime APIs or consume a stream of data by subscribing to it, provided you have the right access.
+
+h3(#producer). Being a producer
+
+A producer of data uses API Streamer to share live data streams with subscribers. Examples of realtime data streams include live sports scores, positioning updates for transport and logistics, or live data streamed from IoT sensors. These data streams can easily be published to the API Streamer via one of your DSN apps, which can then be shared with "data consumers":#consumer. "Learn more about publishing data streams using the API Streamer":/api-streamer/producer.
+
+h3(#consumer). Being a consumer
+
+A consumer is usually invited to API Streamer and granted access to a data stream by a producer. The consumer can subscribe to, manage, and consume the data streams. "Learn more about consuming the data streams using the API Streamer":api-streamer/consumer.
+
+h3(#tutorials). Next up
+
+If you'd like to try out the API Streamer, you can read more about "producers":/api-streamer/producer and "consumers":/api-streamer/consumer or check out our "step-by-step tutorials":todo-link or "video walk-throughs":todo-link to get started quickly.
```

Comment:
I don't think we should ever release code with `todo-link`s.  This is error prone.  Why not state there are tutorials coming soon instead?